js/loader.js

```diff
@@ -91,7 +91,7 @@ var Loader = (function () {
 			var moduleFolder = config.paths.modules + "/" + module;
 
 			if (defaultModules.indexOf(moduleName) !== -1) {
-				moduleFolder = config.paths.modules + "/default/" + module;
```

Comment:
is that path already configurable?

Reply:
if so, why isnt it in the config sample mentioned?

Reply:
(also: maybe we should add that there?)

Reply:
path is already configurable, see defaults.js and I tested this. Found nothing documented about this, can add it to the config sample but I dont't know if this a good idea when users who don't know what they are doing are playing with this variable ...

Reply:
I see. Maybe @MichMich should weigh in on this: if the ALL options should be documented in the config sample?